resources/example_log_lines.ts

```diff
@@ -418,24 +982,71 @@ const exampleLogLines: ExampleLines = {
       en: [
-        '261|2023-04-20T19:04:39.3810000-07:00|Add|400139C5|Type|2|TargetID|10FFFFFF|Name|Omega|MaxHP|8557964|PosX|100|PosY|90|PosZ|-5.456968E-12|Heading|-4.792213E-05|Radius|12.006|BNpcID|3D5C|CurrentMP|10000|MaxMP|10000|Level|90|BNpcNameID|1E0F|WorldID|65535|CurrentWorldID|65535|NPCTargetID|1084E23D|CastDurationMax|-3.689349E+19|e173dbd66eb7c1fe',
-        '261|2023-04-20T19:04:41.9200000-07:00|Change|400139C5|PosX|100.1179|PosY|95.16841|PosZ|-5.456968E-12|Heading|0.08906358|eac28822c9abcde9',
-        '261|2023-04-20T19:06:46.2900000-07:00|Remove|400139C5|09a3165588ea6b13',
+        '261|2023-05-26T21:37:40.5600000-04:00|Add|40008953|BNpcID|3F5A|BNpcNameID|304E|CastTargetID|E0000000|CurrentMP|10000|CurrentWorldID|65535|Heading|-3.1416|Level|90|MaxHP|69200|MaxMP|10000|ModelStatus|18432|Name|Golbez\'s Shadow|NPCTargetID|E0000000|PosX|100.0000|PosY|100.0000|PosZ|0.0300|Radius|7.5000|Type|2|WorldID|65535|9d9028a8e087e4c3',
+        '261|2023-05-26T21:39:41.2920000-04:00|Change|10001234|CurrentMP|2400|Heading|-2.3613|2f5ff0a91385050a',
+        '261|2023-05-26T21:39:42.7380000-04:00|Remove|40008AA0|f4b30f181245b5da',
       ],
     },
+    unitTests: {
+      indexToTest: 0,
+      expectedValues: {
+        type: '261',
+        change: 'Add',
+        id: '40008953',
+        pair: [
+          { key: 'BNpcID', value: '3F5A' },
+          { key: 'BNpcNameID', value: '304E' },
+          { key: 'CastTargetID', value: 'E0000000' },
+          { key: 'CurrentMP', value: '10000' },
+          { key: 'CurrentWorldID', value: '65535' },
+          { key: 'Heading', value: '-3.1416' },
+          { key: 'Level', value: '90' },
+          { key: 'MaxHP', value: '69200' },
+          { key: 'MaxMP', value: '10000' },
+          { key: 'ModelStatus', value: '18432' },
+          { key: 'Name', value: 'Golbez\'s Shadow' },
+          { key: 'NPCTargetID', value: 'E0000000' },
+          { key: 'PosX', value: '100.0000' },
+          { key: 'PosY', value: '100.0000' },
+          { key: 'PosZ', value: '0.0300' },
+          { key: 'Radius', value: '7.5000' },
+          { key: 'Type', value: '2' },
+          { key: 'WorldID', value: '65535' },
+        ],
+      },
+    },
   },
   RSVData: {
     examples: {
       en: [
         '262|2023-04-21T23:24:05.8320000-04:00|en|0000001C|_rsv_32789_-1_1_0_1_SE2DC5B04_EE2DC5B04|Run: ****mi* (Omega Version)|34159b6f2093e889',
         '262|2023-04-21T23:24:05.9210000-04:00|en|00000031|_rsv_3448_-1_1_1_0_S74CFC3B0_E74CFC3B0|Burning with dynamis inspired by Omega\'s passion.|ce9d03bb211d894f',
-        '262|2023-04-21T23:24:06.0630000-04:00|en|00000051|_rsv_35827_-1_1_0_0_S13095D61_E13095D61|Further testing is required.�����,\\r���)������ ��, assist me with this evaluation.|38151741aad7fe51',
+        '262|2023-04-21T23:24:06.0630000-04:00|en|00000051|_rsv_35827_-1_1_0_0_S13095D61_E13095D61|Further testing is required.�����, ���)������ ��, assist me with this evaluation.|38151741aad7fe51',
```

Comment:
This seems like an unintentional change? Did you verify this from raw log line info?

Reply:
> This seems like an unintentional change? Did you verify this from raw log line info?

I originally took it out because it was causing issues with the unit test, and it didn't seem worth it to write logic just to handle special characters solely in a unit test example that we ourselves are providing.  (As an alternative, we could just replace this line with a unit test that doesn't have special characters).  I probably should add a tracking issue to make sure the special character problem is limited to unit testing and not actually a live capture issue.